packages/utils/localization/DateUtils.js

```diff
@@ -0,0 +1,132 @@
+// @flow strict
+
+// test immutability
```

Comment:
test immutability what does this mean? TODO?

Reply:
This was extracted as is from `mobile` repository.

Reply:
Added @TODO